Tests/PointFreeTests/GhostTests.swift

```diff
@@ -0,0 +1,191 @@
+import Either
+import HttpPipeline
+import Models
+import Optics
+@testable import PointFree
+import PointFreePrelude
+import PointFreeRouter
+import PointFreeTestSupport
+import Prelude
+import SnapshotTesting
+import XCTest
+
+final class GhostTests: TestCase {
+  override func setUp() {
+    super.setUp()
+    record=true
+  }
+
+  func testStartGhosting_HappyPath() {
```

Comment:
these tests have some pretty complicated mocking needs in order to get the environment into shape for properly testing ghosting. would love to find ways of simplifying.